File: code/97189.pas
program Bai189;
  var TS, MS : integer;

  function UCLN(m,n: integer): integer;
  begin
    while m <> n  do if m > n then m:= m - n else n := n - m;
    UCLN := m;
  end;
  begin
  readln(TS,MS);
  TS := TS div UCLN(TS,MS);
  MS := MS div UCLN(TS,MS);
  writeln (TS, '/', MS);
end.

File: code/97189.cpp
#include <iostream>
using namespace std;

int ucln(int a,int b){
    while (a!=b){
        if (a>b)
            a=a-b;
        else
            b=b-a;
    }
}

int main() 
{
    int a,b,c;
    cin>>a>>b;
    c=ucln(a,b);
    a=a/c;
    b=b/c;
    cout<<a<<" "<<b;
    return 0;
}